tensorboard/data/server/run.rs

```diff
@@ -269,8 +310,12 @@ fn read_event(
         }
         Some(wt) => wt,
     };
-    if start_time.map(|start| wall_time < start).unwrap_or(true) {
-        *start_time = Some(wall_time);
+    if data
+        .start_time
+        .map(|start| wall_time < start)
+        .unwrap_or(true)
+    {
+        (*data).start_time = Some(wall_time);
```

Comment:
Can just be `data.start_time`; no need to explicitly deref.

(i.e., C’s `operator->` and `operator.` are basically both just `.`.)